From 5aa1b1f48e5cd4186a70b7ce47ae23182e698b65 Mon Sep 17 00:00:00 2001
From: Pavel Tikhomirov <ptikhomirov@virtuozzo.com>
Date: Fri, 18 Feb 2022 18:13:14 +0300
Subject: [PATCH 137/249] zdtm: fix missplacement of err=True

There is no 'err' argument for print(), it should be in grep_errors() in
line below.

Fixes: bed670f62 ("zdtm: print tails of all logs if a test has failed")
Signed-off-by: Pavel Tikhomirov <ptikhomirov@virtuozzo.com>
---
 test/zdtm.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/test/zdtm.py b/test/zdtm.py
index cf73a17ae..daad1b687 100755
--- a/test/zdtm.py
+++ b/test/zdtm.py
@@ -1485,8 +1485,8 @@ class criu:
             self.__page_server_p = None
         if self.__dump_process:
             self.__dump_process.terminate()
-            print("criu dump exited with %s" % self.__dump_process.wait(), err=True)
-            grep_errors(os.path.join(self.__ddir(), "dump.log"))
+            print("criu dump exited with %s" % self.__dump_process.wait())
+            grep_errors(os.path.join(self.__ddir(), "dump.log"), err=True)
             self.__dump_process = None
         if self.__img_streamer_process:
             self.__img_streamer_process.terminate()
-- 
2.35.1

